misk/src/main/kotlin/misk/metrics/MetricsModule.kt

```diff
@@ -1,11 +1,26 @@
 package misk.metrics
 
+import io.prometheus.client.Collector
 import io.prometheus.client.CollectorRegistry
+import io.prometheus.client.hotspot.BufferPoolsExports
+import io.prometheus.client.hotspot.ClassLoadingExports
+import io.prometheus.client.hotspot.GarbageCollectorExports
+import io.prometheus.client.hotspot.MemoryPoolsExports
+import io.prometheus.client.hotspot.StandardExports
+import io.prometheus.client.hotspot.ThreadExports
+import io.prometheus.client.hotspot.VersionInfoExports
 import misk.inject.KAbstractModule
 
 internal class MetricsModule : KAbstractModule() {
   override fun configure() {
     bind<Metrics>().asEagerSingleton()
     bind<CollectorRegistry>().asEagerSingleton()
+    multibind<Collector>().toInstance(StandardExports())
```

Comment:
so we expect other modules to start to bind their own collectors? But these are considered "standard" enough to live here?

Reply:
yea. they are all jvm metrics, so every misk app will want them